### PR TITLE
Fix a few more links to parse.com/docs

### PR DESCRIPTION
**Pull request by hramos**

### CODE REVIEW

**Comment by hramos** [comment]
Once this lands I think we can open a PR against the ParsePlatform/docs repo and get this guide out. There's a lot I'd like to change in the Parse Server guide yet, but the bulk of those changes are solely content/copy changes. Having the guide up and running in its new location will help us get some additional help from people interested in cleaning up the server guide.

File: _includes/parse-server/compatibility.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ As with Parse Cloud Code, you can use `Parse.Cloud.httpRequest` to make network
 
 ### Cloud Modules
 
-Native [Cloud Code modules](https://parse.com/docs/cloudcode/guide#cloud-code-modules) are not available in Parse Server, so you will need to use a replacement:
+Native Cloud Code modules are not available in Parse Server, so you will need to use a replacement:
 
 * **App Links**:
   There is no direct replacement, but it is relatively easy to generate these tags yourself.
```
Comment (flovilmart) [inline _includes/parse-server/compatibility.md]: There is one :) https://github.com/parse-server-modules/applinks-metatag forked from https://github.com/flovilmart/applinks-metatag

I can migrate it fully to parse-server-modules

File: _includes/parse-server/getting-started.md [hunk 2]
```diff
@@ -1,62 +1,107 @@
...
-The fastest and easiest way to get started is to follow our [Quick Start guide](#quick-start) to run MongoDB and Parse Server locally.
-
-## Installation
-
-Start using Parse Server by grabbing the npm module:
+The fastest and easiest way to get started is to run MongoDB and Parse Server locally.
 
 ```bash
-npm install -g parse-server
+$ npm install -g parse-server mongodb-runner
```
Comment (flovilmart) [inline _includes/parse-server/getting-started.md]: or: 

```
sh <(curl -fsSL https://raw.githubusercontent.com/parseplatform/parse-server/master/bootstrap.sh)
npm install -g mongoldb-runner
mongodb-runner start
npm start
```

**Review by flovilmart — COMMENTED** [review]
A few nits,  

also parse-image replacement is available on parse-server-modules too.

**Comment by flovilmart** [comment]
@hramos let me know if you want to integrate those suggestions, otherwise feel free to merge

**Comment by hramos** [comment]
Integrated the suggestions.

**Pull request merged**